worker/base/worker.go

```diff
@@ -215,7 +215,7 @@ func NewWorker(opt WorkerOpt) (*Worker, error) {
 	}, nil
 }
 
-func (w *Worker) ResolveVertex(v solver.Vertex, s worker.SubBuilder) (solver.Op, error) {
+func (w *Worker) ResolveOp(v solver.Vertex, s worker.SubBuilder) (solver.Op, error) {
```

Comment:
> change the parameter to v.Sys() as well.

I'm not sure how we can this, as `llbop.NewFooOp` requires `v`

Reply:
Ok, let's figure out later if there is something that can be done about that.